src/libraries/System.Text.Json/source_generation/src/ReflectionUtils/AssemblyWrapper.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+
+namespace System.Reflection
```

Comment:
Should these reflection types be in System.Text.Json.SourceGeneration.Reflection instead?

Reply:
Yup I'll move them there. We can generalize if other generators need this later.